examples/destructure-tuples/destructure_tuples.bal

```diff
@@ -0,0 +1,25 @@
+import ballerina/io;
+
+type Time [int, decimal];
+
+// This function accepts a variable of type Time and converts it to seconds in decimal.
```

Comment:
```suggestion
// This function accepts a variable of type `Time` and converts it to seconds in `decimal`.
```

We emphasize keywords, type names, variable names, etc. Please check and update wherever applicable.